### ISSUE TITLE
[BUG] Updating Attributes is not idempotent

### ISSUE THREAD

**Issue by bhuddah** [Checkmk/ansible-collection-checkmk.general#886]
Verify first that your issue is not already reported [here](https://github.com/Checkmk/ansible-collection-checkmk.general/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc).
Where possible also test if the latest release and main branch are affected too.
Complete all sections as described!

**Describe the bug**
Updating Attributes via checkmk.general.host is not idempotent.

**Component Name**

Component Name: checkmk.general.host

**Ansible Version**

```console
$ ansible --version
ansible [core 2.17.14]

```

**Checkmk Version and Edition**

```console
Checkmk Raw Edition 2.3.0p34
```

**Collection Version**

```console
$ ansible-galaxy collection list
Collection                      Version
------------------------------- -------
ansible.netcommon               8.1.0
ansible.posix                   2.1.0
ansible.utils                   6.0.0
checkmk.general                 6.2.2
community.crypto                3.0.3
community.general               11.3.0

```

**To Reproduce**
1. run task
2. observe added labels
3. run task again
4. observe that there still are changes reported

**Expected behavior**

When run again, the task does not report changes.
**Actual behavior**

**Minimum reproduction example**

```yaml
# Update only specified attributes
- name: "Update only specified attributes"
  checkmk.general.host:
    server_url: "http://myserver/"
    site: "mysite"
    automation_user: "myuser"
    automation_secret: "mysecret"
    name: "my_host"
    update_attributes: |
      {
      "labels": { "thisisatest": "this is a test" }
      }
    state: "present"
```

**Additional context**

**Comment by lgetwan** [comment]
Hi @bhuddah,
I will have a look at this as soon as possible. Thanks  for reporting!
Lars

**Comment by msekania** [comment]
@bhuddah ,

```yaml
- name: "Update only specified attributes"
  checkmk.general.host:
    server_url: "http://myserver/"
    site: "mysite"
    automation_user: "myuser"
    automation_secret: "mysecret"
    name: "my_host"
    update_attributes:
      labels: { "thisisatest": "this is a test" }
    state: "present"
```

**works and is idempotent!**

ansible [core 2.17.13]
Checkmk  CEE Edition 2.4.0p12

might be an issue with jinja2 pipe ('|') filter
```yaml
    update_attributes: |
      {
      "labels": { "thisisatest": "this is a test" }
      }
```
which does not work in my case at all
